cms/envs/production.py

```diff
@@ -554,6 +556,13 @@ def get_env_setting(setting):
 if FEATURES.get('ENABLE_CORS_HEADERS'):
     CORS_ALLOW_CREDENTIALS = True
     CORS_ORIGIN_WHITELIST = ENV_TOKENS.get('CORS_ORIGIN_WHITELIST', ())
+
+    # values are already updated above with default CORS_ORIGIN_WHITELIST values but in
+    # case of new version django_cors_headers they will get override.
+    cors_major_version = int(version('django_cors_headers').split('.')[0])
+    if cors_major_version >= 3 and ENV_TOKENS.get('CORS_ORIGIN_WHITELIST_WITH_SCHEME'):
```

Comment:
I'm tempted to suggest moving this logic into a shared function instead of repeating it in 3 settings files, but if we delete it right after the upgrade, that might not be worth the effort.

Reply:
Its a temporary function and will be remove after deployment. Eventually we will the `CORS_ORIGIN_WHITELIST_WITH_SCHEME` also.

Reply:
Can we add a default to that get or at least condition this if statement of `if CORS_ORIGIN_WHITELIST`.

Reply:
```suggestion
    if cors_major_version >= 3 and ENV_TOKENS.get('CORS_ORIGIN_WHITELIST_WITH_SCHEME', ()):
```

Reply:
or 
```suggestion
    if cors_major_version >= 3 and CORS_ORIGIN_WHITELIST and ENV_TOKENS.get('CORS_ORIGIN_WHITELIST_WITH_SCHEME'):
```